src/main/java/org/fundacionjala/at15/pokemon/io/Writer.java

```diff
@@ -22,9 +22,9 @@ public static void writeToJson(Entity entity) {
         }
     }
 
-    private static void checkFolder(File path) {
+    public static void checkFolder(File path) {
         if (path.exists()) {
-            System.out.println(path + " already exists");
+            System.out.println("Data saved!");
```

Comment:
This message could be confusing for the user because this verifies only if the directory already exists, showing "Data saved" message could suggest the user that a file was saved.

Reply:
The previous message i belive was more confusing for the user. Data saved! is more clear that the folder is updated